custom/icds/messaging/custom_recipients.py

```diff
@@ -32,3 +33,9 @@ def supervisor_from_awc_owner(case_schedule_instance):
         return None
 
     return awc.parent
+
+
+def parent_of_ccs_record(case_schedule_instance):
+    if not case_schedule_instance.case:
+        return None
+    return mother_person_case_from_ccs_record_case(case_schedule_instance.case)
```

Comment:
this could raise a `CaseRelationshipError` - we should catch it and when it happens we should notify exception and return None.

returning None will make the framework log an error for it in the messaging history models (no recipient error). the point of notifying the exception will be to look into any data integrity issues, but we don't want to fail hard because there's nothing the framework can do to correct this